### ISSUE TITLE
Painting template; white turns black after saving the project

### ISSUE THREAD

**Issue by kkallio** [OpenOrienteering/mapper#1103]
### Steps to reproduce
1. Paint template with white colour
2. Save the project
3. Close the application
4. Open the application
5. Open the saved project

### Actual behaviour
Area painted with white has turned black

### Expected behaviour
White should stay white after saving and closing

### Configuration
Mapper Version: 0.7.0, 0.8.1
Operating System: Windows 10 Pro 1803

**Comment by kkallio** [comment]
Yes, that's right. I converted the template from jpg to png and now the white stays white. Too bad I did't know it, caused some extra work.

**Comment by dg0yt** [comment]
I don't know the implementation details. From my usage, I would says "white" means erasing which means transparency, and JPEG doesn't support transparency.

**Comment by lpechacek** [comment]
An observation. The white (eraser) color is implemented as drawing with `QColor(255, 255, 255, 0)` (full white, zero opacity). The result in the template is a white area. However, changing the color in the code to `QColor(255, 255, 255, 1)` (full white, a tiny bit opaque but practically transparent) results in nothing being drawn on the template.

At this point, I'm wondering what I'm missing. A fully transparent "color" leaves white spots while marginally less transparent color leaves no traces. How come?
